src/model/userdbmapper.js

```diff
@@ -0,0 +1,59 @@
+import { ref, set, remove } from "firebase/database";
+
+/**
+ * Add user to firebase database
+ * @param {Firebase database} database 
+ * @param {Id of the user} userId 
+ * @param {Name of the user} name 
+ * @param {E-Mail of the user} email 
+ * @param {Url to profile pictore of the user} imageUrl 
+ */
+const writeUserData = function(database, userId, name, email, imageUrl) 
+{
+  set(ref(database, 'users/' + userId), {
+    username: name,
+    email: email,
+    profile_picture : imageUrl
+  })
+}
+
+/**
+ * Delete the user account in the firebase database and the firebase user authentication.
+ * @param {Firebase instance} firebase 
+ * @param {Firebase database} database 
+ */
+const deleteAccount = function(firebase, database) 
+{
+   var user = firebase.auth().currentUser;
+   // delete user in database
+   remove(ref(database, 'users/' + user.uid)).then(function(error)
+   {
+       if (error)
+       {
+           console.log(error.message)
+       }
```

Comment:
Syntax-Aussehen ist hier anders als in anderen files, z.B in checkTitle in itemmodel.
```
if (title.length > 255) {
      throw new Error('too many characters')
    }
```
D.h. es sollte so geschrieben werden: 
```
if (error) {
  console.log(error.message)
}
```
Und das ist auch an anderen Stellen. Vermutlich wäre es am Besten/Einfachsten an dieser Stelle prettier in unser Projekt einzuführen. Dann hätten wir ein einheitliches Aussehen vom code. Was meinst du? Wir könnten den PR dann mal so mergen und ich könnte in einem PR danach die index.js und prettier reinbringen.

Reply:
Yes, I agree. There is a mix of different error handling. If you would like to introduce prettier, feel free to do so in your new branch ;)